packages/plugins/Solana/package.json

```diff
@@ -4,6 +4,8 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "dependencies": {
+    "@bonfida/name-offers": "^0.0.1",
+    "@bonfida/spl-name-service": "^0.1.27",
```

Comment:
What's the difference between `@bonfida/spl-name-service` and `@solana/spl-name-service`?

Reply:
reverse domain api in @bonfida/spl-name-service